packages/ui/readme.md

```diff
@@ -1,28 +1,230 @@
 # Penumbra UI
 
-This package contains the UI components for the Penumbra web.
+The Penumbra UI library is a set of UI components purpose-built for the Penumbra ecosystem. Use these components to get rendering of various Penumbra data types out of the box, and to create a UI that is consistent with other Penumbra UIs' look and feel.
 
 ## Set up
 
 First, install the library:
 
 ```bash
-npm i @penumbra-zone/ui
+npm install @penumbra-zone/ui
 ```
 
-Then, configure the Tailwind in your project. Edit `tailwind.config.js`:
+Then, use components by importing them from their specific files:
 
-```js
-export default {
-  content: [
-    // Parses the classes of the UI components
-    './node_modules/@penumbra-zone/ui/**/*.js',
-  ],
-};
+```tsx
+import ValueViewComponent from '@penumbra-zone/ui/ValueViewComponent';
 ```
 
-Finally, import the library CSS to the entry point of your app:
+Deprecated components can be imported from `@penumbra-zone/ui/components/ui/<component-name>`, where `<component-name>` should be replaced with the kebab-cased component name.
 
-```js
-import '@repo/ui/styles/globals.css';
+## Development
+
+These guidelines are for maintainers of the components in the Penumbra UI library.
+
+### Components must be located at `./components/v2/<ComponentName>/index.tsx`.
+
+This ensures that the Penumbra UI `package.json` `exports` field works as intended, so that components can be imported via `@penumbra-zone/ui/<ComponentName>`.
+
+Note that `<ComponentName>` should be replaced with the `UpperCamelCase` component name — e.g., `./components/v2/LoadingIndicator/index.tsx`.
```

Comment:
As long as people import it using the same capitalization at the file name, it should work across all systems, right? (I wanted to do this change both A) to make imports a little less annoying to type, and B) because it seems to be the norm across other libraries like MUI — [example](https://mui.com/system/react-box/#basics).)